src/index.js

```diff
@@ -1,14 +1,21 @@
 // @flow
 import SwaggerUIStandalonePreset from "swagger-ui";
-import "swagger-ui/dist/swagger-ui.css";
+import "../node_modules/cookieconsent/build/cookieconsent.min.js";
+import dotenv from "dotenv";
 
+import initializeCookieConsentBar from "./helpers/tracking";
 import registerServiceWorker from "./helpers/registerServiceWorker";
+
+import "swagger-ui/dist/swagger-ui.css";
+
 import {
     BunqLayoutPlugin,
     DisableTryItOutPlugin
 } from "./plugins";
 import "./scss/index.css";
 
+dotenv.config();
```

Comment:
can you do this below with the other initializers?

Reply:
it's recommended to do it as soon as possible, in case any of the Swagger extensions use .env

Reply:
ack